lib/rules/jsx-no-literals.js

```diff
@@ -40,116 +167,267 @@ module.exports = {
 
     messages,
 
-    schema: [{
-      type: 'object',
-      properties: {
-        noStrings: {
-          type: 'boolean',
-        },
-        allowedStrings: {
-          type: 'array',
-          uniqueItems: true,
-          items: {
-            type: 'string',
+    schema: [
+      {
+        type: 'object',
+        properties: Object.assign(
+          {
+            elementOverrides: {
+              type: 'object',
+              patternProperties: {
+                [overridableElementPattern]: {
+                  type: 'object',
+                  properties: Object.assign(
+                    {
+                      applyToNestedElements: {
+                        type: 'boolean',
+                      },
+                    },
+                    commonPropertiesSchema
+                  ),
+
+                },
+              },
+            },
           },
-        },
-        ignoreProps: {
-          type: 'boolean',
-        },
-        noAttributeStrings: {
-          type: 'boolean',
-        },
+          commonPropertiesSchema
```

Comment:
This could probably be handled with JsonSchema's `definitions`/`$refs` properties but I couldn't figure out how to best accomplish it.

Reply:
maybe, but it's clearer to avoid it i think.